db/seeds/object_manager_attributes.rb

```diff
@@ -323,6 +322,45 @@
   position: 900,
 )
 
+ObjectManager::Attribute.add(
+    force: true,
+    object: 'Ticket',
+    name: 'organization_id',
+    display: 'Organization',
+    data_type: 'select_organization',
+    data_option: {
+        relation: 'Organization',
+        nulloption: false,
+        multiple: false,
+        null: false,
+        default: 0,
+        translate: true
+    },
+    editable: false,
+    active: true,
+
+    screens: {
+        create_middle_org: {
```

Comment:
See first comment.

Reply:
Uniq screen for new field as 'create_middle', 'create_top' and etc.

Reply:
See comments above regarding generic hooks.

Reply:
If I would use generic hooks then on ticket create form there appear wrong additional fields with the same hooks.

Reply:
Then we need to change the `priority` of those so that the order matches the expectations.